### PR TITLE
feat(TCK): Add `AccountUpdate` method

### LINKED ISSUE #1018: `TCK-Go`: Implement JSON-RPC methods for `AccountUpdateTransaction`
### Problem

We should implement JSON-RPC methods (endpoints) for AccountUpdateTransaction for the Go TCK server

### Solution

Add the following methods:
- Account: updateAccount

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by 0xivanov**
**Description**:
`AccountUpdate` for the TCK server.

**Related issue(s)**:

Fixes #1018

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: tck/methods/key.go [hunk 1]
```diff
@@ -60,26 +60,34 @@ func processKeyRecursively(params param.KeyParams, response *response.GenerateKe
 
 	case param.ED25519_PUBLIC_KEY, param.ECDSA_SECP256K1_PUBLIC_KEY:
 		var publicKey string
+		var privateKey string
```
Comment (0xivanov) [inline tck/methods/key.go:63]: can be optimised

File: tck/methods/account.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ func (a *AccountService) CreateAccount(_ context.Context, accountCreateParams pa
 		}
 		transaction.SetKey(key)
 	}
+
```
Comment (0xivanov) [inline tck/methods/account.go:38]: revert the new lines

**Review by 0xivanov — COMMENTED** [review]
Don't merge until comments in TCK repo are fixed

**Review by gsstoykov — APPROVED** [review]
LGTM.

**Review by gsstoykov — APPROVED** [review]

**Pull request merged**